### PR TITLE
#13 provide customizable swipeble color

### PR DESCRIPTION
**Pull request by dbenchi**

### CODE REVIEW

File: lib/awesome-angular-swipe.js [hunk 1]
```diff
@@ -128,5 +130,5 @@
       }]);
   }
 
-  awesomeAngularSwipe(angular, (typeof global !== 'undefined' ? global : window).interact);
+  awesomeAngularSwipe(angular, (typeof global !== 'undefined' ? global : window).swipe);
```
Comment (rpignolet) [inline lib/awesome-angular-swipe.js]: If I'm not wrong, the function `awesomeAngularSwipe` takes only one parameter.

File: README.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ It works only for mobile device. It will do nothing on desktop (except using the
 You might want to take a look at the index.html file inside the *example* folder.
 The directive *swipe* can be used as a class or as an attribute. Then inject callbacks on right and left swipe, and template customization like so:
 
-    <div class="swipe" swipe-left="doSomething" swipe-right="doAnotherSomething" left-template="leftTemplate.html" right-template="rightTemplate.html"
+    <div class="swipe" swipe-left="doSomething" swipe-right="doAnotherSomething" left-template="leftTemplate.html" right-template="rightTemplate.html" swipeable-color="#fff">
       Swipe me!
     </div>
 
-Note that *swipe-left* and *swipe-right* are functions, *left-template* and *right-template* are whatever templates you want.
+Note that *swipe-left* and *swipe-right* are functions, *left-template* and *right-template* are whatever templates you want, and swipeable-color is the color of the swipeable block (default value = '#fff')
```
Comment (thomas-hilaire) [inline README.md]: Wrap swipeable-color with *, as it is for other attributes

File: lib/awesome-angular-swipe.js (hunk 1 above)
Comment (dbenchi) [inline lib/awesome-angular-swipe.js]: Done

File: README.md (hunk 2 above)
Comment (dbenchi) [inline README.md]: Done

**Pull request merged**